### ISSUE TITLE
Build of experimental.pyx fails when CUDA is disabled

### ISSUE THREAD

**Issue by kohr-h** [astra-toolbox/astra-toolbox#21]
I get the following compile error in the Cython part:
```
gcc [snip] ... [/snip] -c astra/experimental.cpp -o build/temp.linux-x86_64-3.4/astra/experimental.o
astra/experimental.cpp: In function ‘PyObject* __pyx_pf_5astra_12experimental_do_composite(PyObject*, PyObject*, PyObject*, PyObject*, PyObject*)’:
astra/experimental.cpp:884:3: error: ‘CCompositeGeometryManager’ is not a member of ‘astra’
   astra::CCompositeGeometryManager __pyx_v_m;
   ^
astra/experimental.cpp:1224:21: error: ‘__pyx_v_m’ was not declared in this scope
     __pyx_t_7 = ((!(__pyx_v_m.doFP(__pyx_v_projector, __pyx_v_vol, __pyx_v_proj) != 0)) != 0);
                     ^
astra/experimental.cpp:1267:21: error: ‘__pyx_v_m’ was not declared in this scope
     __pyx_t_7 = ((!(__pyx_v_m.doBP(__pyx_v_projector, __pyx_v_vol, __pyx_v_proj) != 0)) != 0);
                     ^
error: command 'gcc' failed with exit status 1
```
The interesting thing about this failure is that it did not occur before 12:30 today (Jan 18) since we have successful builds from then. I don't see any related commits on your side, though.

I'm using GCC 5.3, may that be a problem? My manually specified compiler is not used by Cython, btw :wink:

**Comment by wjp** [comment]
Could you copy-paste the full compile command?

**Comment by wjp** [comment]
And are you building with CUDA support? Come to think of it, this experimental.pyx should probably be disabled entirely if CUDA is disabled.

**Comment by kohr-h** [comment]
You're totally right, here's what happened: I had a workaround for the `NVCC_FLAGS` thing using `sed`, which became obsolete after your upstream fix, then `nvcc` said "no" again, the whole thing got built without CUDA, and then the `experimental.pyx` build failed. So it's probably due to disabled CUDA.

**Comment by dmpelt** [comment]
I could reproduce this and created a PR with a fix at #22 .

**Comment by wjp** [comment]
Should be ok now.
